lib/CModHandler.cpp

```diff
@@ -91,26 +89,26 @@ void CIdentifierStorage::requestIdentifier(ObjectCallback callback)
 		resolveIdentifier(callback);
 }
 
-void CIdentifierStorage::requestIdentifier(std::string scope, std::string type, std::string name, const std::function<void(si32)> & callback)
+void CIdentifierStorage::requestIdentifier(std::string scope, std::string type, const std::string & name, const std::function<void(si32)> & callback)
 {
 	auto pair = vstd::splitStringToPair(name, ':'); // remoteScope:name
 
-	requestIdentifier(ObjectCallback(scope, pair.first, type, pair.second, callback, false));
+	requestIdentifier(ObjectCallback(std::move(scope), pair.first, std::move(type), pair.second, callback, false));

```

Comment:
IMO passing by reference is more clean code than forwarding it via std::move & less prone to errors after changes

(same applies to other similar changes in this file)

Reply:
done